### PR TITLE
Fixes #15544: show a loading screen while resource is loading.

### PR DESCRIPTION
**Pull request by waldenraines**
Add support for a loading screen into the nutupane layout so that
a loading screen can be shown while the resource is loading.

http://projects.theforeman.org/issues/15544

### CODE REVIEW

File: app/assets/javascripts/bastion/layouts/panel.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="loading-mask loading-mask-panel fa-3x" ng-show="panel.loading">
+    <i class="fa fa-spinner fa-spin"></i>
+    {{ "Loading..." | translate }}
```
Comment (ehelms) [inline app/assets/javascripts/bastion/layouts/panel.html]: Nitpick indentation is off.

**Comment by ehelms** [comment]
This probably ought to be a component given it deals with and makes assumptions about data being present; however, I won't force that change.

**Comment by waldenraines** [comment]
> This probably ought to be a component given it deals with and makes assumptions about data being present; however, I won't force that change.

I was following the pattern of the nutupane layouts.

This all will need to be reworked anyway for non-nutupane :smile:

**Comment by ehelms** [comment]
ACK

**Pull request merged**